packages/fewer/src/Repository/index.ts

```diff
@@ -51,7 +53,7 @@ export class Repository<
       LoadAssociations & JoinAssociations
     > {
   // Stash the schema type so that the generic can be used as a type constraint.
```

Comment:
If this is consumed outside of this file now, you can probably remove this comment because it's clear through our use of `INTERNAL_TYPES`.